py/openage/convert/mediafile.py

```diff
@@ -77,6 +136,10 @@ def media_convert(args):
         args.extract.append('*:*.*')
 
     extraction_rules = [ExtractionRule(e) for e in args.extract]
+    if args.jobs == 0:
+        args.jobs = os.cpu_count()
```

Comment:
You could set `default=os.cpu_count()` in the argparse definition. Setting it to 0 as default, dunno :)